### PR TITLE
feat(24.04): openjdk-11 headless slices

### PR DESCRIPTION
**Pull request by vpa1977**
# Proposed changes

Add openjdk-11 slices except AWT slice.

## Checklist

* [x] I have read the [contributing guidelines](
https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
* [x] I have tested my changes ([see how](https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md#7-test-your-slices-before-opening-a-pr))
* [x] I have already submitted the [CLA form](
https://ubuntu.com/legal/contributors/agreement)

### CODE REVIEW

**Comment by vpa1977** [comment]
Needs arm64 pipeline retry

**Review by cjdcordeiro — APPROVED** [review]
thanks for the changes. LGTM

**Comment by vpa1977** [comment]
Note: dropped jvm.cfg-default https://github.com/canonical/chisel-releases/pull/363

File: slices/openjdk-11-jdk-headless.yaml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    contents:
+      /usr/lib/jvm/java-11-openjdk-*/bin/javac:
+      /usr/lib/jvm/java-11-openjdk-*/bin/javadoc:
+      /usr/lib/jvm/java-11-openjdk-*/bin/javap:
+      /usr/lib/jvm/java-11-openjdk-*/bin/jdeprscan:
+      /usr/lib/jvm/java-11-openjdk-*/bin/jdeps:
+      /usr/lib/jvm/java-11-openjdk-*/bin/jimage:
+      /usr/lib/jvm/java-11-openjdk-*/bin/serialver:
+
+  # excludes jaotc - this requires binutils
```
Comment (rebornplusplus) [inline slices/openjdk-11-jdk-headless.yaml:24]: Feel free to raise a PR or issue to add `binutils` slice. :)

File: slices/openjdk-11-jdk-headless.yaml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  standard:
+    essential:
+      - openjdk-11-jdk-headless_core
+      - openjdk-11-jre-headless_standard
```
Comment (rebornplusplus) [inline slices/openjdk-11-jdk-headless.yaml:28]: Nitpick: just the `standard` is fine.

**Review by rebornplusplus — APPROVED** [review]
Looks a good to me, thanks for the superb work here!

Just a comment and a nitpick below. Feel free to ignore both. :)

File: slices/openjdk-11-jdk-headless.yaml (hunk 2 above)
Comment (vpa1977) [reply slices/openjdk-11-jdk-headless.yaml:28]: This is core *jdk* + standard *jre*, so we need both.

Comment (rebornplusplus) [reply slices/openjdk-11-jdk-headless.yaml:28]: Oh, right! :eyeglasses:

**Pull request merged**